src/components/one-time-donation/components/AnonymousForm.tsx

```diff
@@ -0,0 +1,68 @@
+import { Box, Collapse, Grid, Typography } from '@mui/material'
+import * as React from 'react'
+import FormTextField from 'components/common/form/FormTextField'
+import CircleCheckboxField from 'components/common/form/CircleCheckboxField'
+import { useField } from 'formik'
+import { useTranslation } from 'next-i18next'
+import { ExpandLess, ExpandMore } from '@mui/icons-material'
+
+export default function AnonymousMenu() {
```

Comment:
We may need to choose between `AnonymousForm` (filename) or `AnonymousMenu` (component name) and stick with one of those

Reply:
```suggestion
export default function AnonymousForm() {
```